packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Resizer.ts

```diff
@@ -111,27 +124,32 @@ export function doubleCheckResize(
  * @internal
  * Get HTML for resize handles at the corners
  */
-export function getCornerResizeHTML({
-    borderColor: resizeBorderColor,
-    handlesExperimentalFeatures: handlesExperimentalFeatures,
-}: ImageHtmlOptions): CreateElementData[] {
+export function getCornerResizeHTML(
+    {
+        borderColor: resizeBorderColor,
+        handlesExperimentalFeatures: handlesExperimentalFeatures,
+    }: ImageHtmlOptions,
+    onShowResizeHandle?: OnShowResizeHandle
+): CreateElementData[] {
     const result: CreateElementData[] = [];
 
     Xs.forEach(x =>
-        Ys.forEach(y =>
-            result.push(
-                (x == '') == (y == '')
-                    ? getResizeHandleHTML(
-                          x,
-                          y,
-                          resizeBorderColor,
-                          handlesExperimentalFeatures
-                              ? HandleTypes.CircularHandlesCorner
-                              : HandleTypes.SquareHandles
-                      )
-                    : null
-            )
-        )
+        Ys.forEach(y => {
+            let elementData = (x == '') == (y == '')
+            ? getResizeHandleHTML(
+                  x,
+                  y,
+                  resizeBorderColor,
+                  handlesExperimentalFeatures
+                      ? HandleTypes.CircularHandlesCorner
+                      : HandleTypes.SquareHandles
+              )
+            : null;
+            if (onShowResizeHandle) {
+                onShowResizeHandle(elementData, x, y)
```

Comment:
I think you need one more parameter to tell if this elementData is for resizng, rotating, or cropping.

Reply:
Unless, you'd only like to support this for resize, and I think that is fine.

Reply:
Yes, I'm only going to support Resize.